ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -146,6 +137,26 @@ class MSFListCellStateImpl: NSObject, ObservableObject, Identifiable, MSFListCel
         }
     }
 
+    @Published var leadingViewSize: MSFListCellLeadingViewSize = .medium {
+        didSet {
+            if leadingViewSize != oldValue {
```

Comment:
Can you switch this to guard?

Reply:
Is that the pattern we want going forward? I did a quick search through the codebase and found around 90 instances of using `if oldValue != foo` or `if foo != oldValue`, but only around a dozen or so using `guard`.

My personal bias is that the `if` version is more readable when we're just comparing against `oldValue`, but I'm more curious what the approach should be going forward.

Reply:
We can slowly convert all early return from "if" to "guard" as we touch the files.

The rationale behind this is in the swift-guide (early return section):
https://github.com/microsoft/swift-guide/blob/gh-pages/EarlyReturn.md